test/unit-tests.test.ts

```diff
@@ -117,10 +117,37 @@ test('cannot use var', () => {
     ]));
 });
 
-test('cannot use switch', () => {
-  expect(staticError(`switch (5) { case 5: }`)).toEqual(
+test('no switch case fall-through', () => {
+  //Empty case.
+  expect(staticError(`let x = 1;
+    switch (x) {case 1: case 2: break; default: x += 2;}`
+  )).toEqual(
     expect.arrayContaining([
-      expect.stringMatching(`Do not use the 'switch' statement.`)
+      expect.stringMatching(`No case fall-through.`),
```

Comment:
Let's not use jargon, keeping in mind that it is meant for intro programmers. 
`Every case in a switch block must close with a break statement`
Or something similar along the lines. If the error message can convey that it is an enforced restriction (not a missing feature), all the better.

Reply:
I made an effort to provide user-friendly messages.